tools/wpt/browser.py

```diff
@@ -134,15 +139,15 @@ def install(self, dest=None, channel="nightly"):
             if platform == "mac" and os.path.exists(os.path.join(dest, "Firefox Nightly.app")):
```

Comment:
Don't we want this to be application_name[channel] too?

Reply:
Yep! Missed that earlier. Thanks for pointing it out :)